cunumeric/module.py

```diff
@@ -1314,33 +1361,45 @@ def _concatenate(
     dtype=None,
     casting="same_kind",
     common_info=None,
+    slicing_only=False,
```

Comment:
Do you actually have any test that passes `slicing_only=True`? If we don't test this regularly, this can easily get bit rotted.